lib/bsdlib/Kconfig

```diff
@@ -5,6 +5,12 @@
 
 menu "BSD Library for nrf91"
 
+config PM_PARTITION_SIZE_BSDLIB_RAM
```

Comment:
@tejlmand  Do you think we should collect all of these configurations under the "partition manager" configuration? Like we did with the `pm.yml` files themself?

Reply:
I think that grouping settings related to partitioning close together makes sense.
Cause if you change this setting, then it will affect settings of the other partitions, and having them close togther will make it easier to find.